mix.exs

```diff
@@ -0,0 +1,43 @@
+defmodule PulsarClientErl.MixProject do
+  use Mix.Project
+
+  def project() do
+    [
+      app: :pulsar,
+      version: read_version(),
+      elixir: "~> 1.13",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  def application do
+    [
+      extra_applications: [:logger],
+      mod: {:pulsar_app, []}
+    ]
+  end
+
+  defp deps do
+    [
+      {:crc32cer, github: "emqx/crc32cer", tag: "0.1.9"},
+      {:murmerl3, github: "emqx/murmerl3", ref: "5ae165c034e601b52e9569187b33ac95f6b3878e"},
+      {:snappyer, github: "emqx/snappyer", tag: "1.2.5"},
+      {:replayq, github: "emqx/replayq", tag: "0.3.4"},
+      {:redbug, "~> 2.0"}
```

Comment:
redbug should be removed?

Reply:
or maybe just a dev or test dependency

Reply:
Since mix is used here just for ease of development, I think we can keep it.  I'll change it to only test or dev envs.

Reply:
:heavy_check_mark: